test/volume_cb_hpolytope.cpp

```diff
@@ -5,7 +5,9 @@
 
 // Licensed under GNU LGPL.3, see LICENCE file
 
-#include "doctest.h"
```

Comment:
you should include this, otherwise tests are failing

Reply:
Done!

Reply:
tests are passing now, great!